authmanager/authmanager.go

```diff
@@ -59,6 +59,7 @@ func acquireToken(authCfg *config.Auth, tokenKey string, instructions string) er
 		return fmt.Errorf("couldn't acquire token: %w", err)
 	}
 
+	fmt.Println()
```

Comment:
this is so that there is a newline after pasting the api token on the terminal.

Otherwise, it looks something like this:
```
please paste your token below:
Token: ERROR: main.go:53: // TODO foo: invalid
        > todo should look like ...
```